src/Client.php

```diff
@@ -53,8 +56,9 @@ class Client
      * @param GuzzleClient|null $client
      * @param int $maxChunkSize Set max chunk size per request (determines when to switch from "one shot upload" to upload session and defines chunk size for uploads via session).
      * @param int $maxUploadChunkRetries How many times to retry an upload session start or append after RequestException.
+     * @param string $teamMemberID The team member ID to be specified for Dropbox business accounts
      */
-    public function __construct($accessTokenOrAppCredentials = null, GuzzleClient $client = null, int $maxChunkSize = self::MAX_CHUNK_SIZE, int $maxUploadChunkRetries = 0)
+    public function __construct($accessTokenOrAppCredentials = null, GuzzleClient $client = null, int $maxChunkSize = self::MAX_CHUNK_SIZE, int $maxUploadChunkRetries = 0, string $teamMemberId = '')
```

Comment:
Use `null` as the default.

Reply:
It was null in my first commit and then chose to make it an empty string... Well, glad it's been accepted.
Sorry to ask but, I'm not exactly a github expert... Should I do something or you did it yet?

Reply:
Done, it was my initial choice in my first commit, I then changed it to an empty string. Oh well. Glad it's accepted!